### DISCUSSION TITLE
Can't install

### DISCUSSION THREAD

**Discussion by youngvernon** [SonicJs-Org/sonicjs#106] [Q&A] [answered]
Just tried to build and start project on a fresh install of Debian 11 and tried under Windows 11 but setup fails. Please list the actual requirements to get things up and going such as node.js version, python, yarn, database or whatever else is unique to your build environment because it's not as simple as you claimed to be in documentation or readme.

**Comment by lane711** [comment, marked as answer]
This issues is resolved per thread in the Issues section
